src/utils/viewer/viewer.cpp

```diff
@@ -465,15 +473,18 @@ void Viewer::wiggleLastObject() {
 }
 
 void Viewer::toggleNavMeshVisualization() {
```

Comment:
Would be nice if we didn't have to duplicate navMeshVisPrimID_, navMeshVidNode_, and toggleNavMeshVisualization in Viewer and Simulator.

Reply:
Agreed. We need to refactor Viewer to use the Simulator class and this kind of duplication won't be a problem anymore. 🤷‍♂️

Reply:
I know I'm late to this conversation, but completely agree.